src/components/portrait/portrait.component.tsx

```diff
@@ -23,7 +25,9 @@ export type PortraitSizes = "XS" | "S" | "M" | "ML" | "L" | "XL" | "XXL";
 
 let deprecatedGravatarWarnTriggered = false;
 
-export interface PortraitProps extends MarginProps {
+export interface PortraitProps
+  extends MarginProps,
+    Omit<TagProps, "data-component"> {
```

Comment:
question: is these change within the scope of this commit (i.e. upgrading React to v18) as well?

Reply:
This change isn't strictly required for the v18 upgrade, but I included it for consistency with other components while reviewing typing. Let me know if you'd prefer to revert it.

Reply:
Again, a separate commit would be good I think